src/components/Contact/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import "./contact.scss";
+import { TiSocialFacebookCircular } from 'react-icons/ti';
+import { TiSocialTwitterCircular } from 'react-icons/ti';
+import { TiSocialGooglePlusCircular } from 'react-icons/ti';
+import logo from '../../assets/img/Renome.svg';
+
+const Contact = () => {
+    return (
+        <div className="contact">
+            <h2 className="contact__title">follow us:</h2>
+            <div className="contact__icons">
+                <TiSocialFacebookCircular className="contact__icons__icon" />
+                <TiSocialTwitterCircular className="contact__icons__icon" />
+                <TiSocialGooglePlusCircular className="contact__icons__icon" />
```

Comment:
By clicking on each sociality can open the home page for them in a new tab? If not, then please add functionality by clicking open new tab with the home page for each sociality.